static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -465,6 +471,7 @@ class GroupDetails extends Component<Props, State> {
           Sentry.captureException(new Error('Project not found'));
         });
       } else {
+        markEventSeen(api, params.orgId, project.slug, params.groupId);
```

Comment:
@ryan953 can you fix this error?

Reply:
done

had tried to quickly resolve conflicts in the web editor, but it's not the same :(